submit_to_statuspage.io.py

```diff
@@ -0,0 +1,22 @@
+import json
+import httplib, urllib, time
+
+from pprint import pprint
+
+with open('/milklevel.json') as milk:
+  data = json.load(milk)
+
+api_key = 'REDACTED'
+page_id = 'REDACTED'
+metric_id = 'REDACTED'
```

Comment:
How will these be provided?

Reply:
Once it's on the milk maid, we can manually change the values. There is no puppet or anything automated here.